lucet-runtime/lucet-runtime-internals/src/c_api.rs

```diff
@@ -225,7 +225,7 @@ pub mod lucet_state {
                     val: lucet_state_val {
                         fault: lucet_runtime_fault {
                             fatal: details.fatal,
-                            trapcode: details.trapcode.into(),
+                            trapcode: (&details.trapcode).into(),
```

Comment:
Nope. I refactored the Rust trapcode to be an Option<TrapCode>, but since C doesn't have a reasonable way to do generic option types, the `lucet_trapcode` enum has a variant `unknown` to represent the None case.

Reply:
I don't understand; if I take out the extra parens and explicit `&`, everything works the same for me.

Reply:
oh i thought you mean the `into` was a no-op. sorry, my bad.